### PR TITLE
Mark EIP-3525 as in review status

### PR DESCRIPTION
**Pull request by will-edge**
Mark EIP-3525 as in review status

### CODE REVIEW

File: EIPS/eip-3525.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ eip: 3525
 title: Semi-Fungible Token Standard
 author: Will Wang (@will-edge), Mike Meng <myan@solv.finance>, Ethan Y. Tsai <yee.tsai@gmail.com>, Ryan Chow <ryanchow@solv.finance>, Zhongxin Wu <wuzhongxin@solv.finance>, AlvisDu <alvis@solv.finance>
```
Comment (Pandapip1) [inline EIPS/eip-3525.md:4]: Nit: would prefer more GitHub usernames

File: EIPS/eip-3525.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ requires: 165, 721
 
 ## Simple Summary
 
-This is a standard for semi-fungible tokens. The set of smart contract interfaces described in this document defines an ERC-721 extension/compatible token standard. This standard introduces an <ID, SLOT, VALUE> triple scalar model that represents the semi-fungible structure of a token. It also introduces new transfer models as well as approval models that reflect the semi-fungible nature of the tokens.
+This is a standard for semi-fungible tokens. The set of smart contract interfaces described in this document defines an ERC-721 compatible token standard. This standard introduces an <ID, SLOT, VALUE> triple scalar model that represents the semi-fungible structure of a token. It also introduces new transfer models as well as approval models that reflect the semi-fungible nature of the tokens.
```
Comment (Pandapip1) [inline EIPS/eip-3525.md:15]: Simple summary is no longer a valid section name. Consider shortening and putting it in the description field.

File: EIPS/eip-3525.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ requires: 165, 721
...
 
 ## Abstract
```
Comment (Pandapip1) [inline EIPS/eip-3525.md:18]: Making comments here, as I cannot suggest edits to un-modified lines:

- Throughout: Currently, EIPs with the ERC category are still referred to as EIPs. I would hold off on this though, as #5190 is in the works.
- L19: A relative link to EIP-721 needs to be added.
- L35-L39: No external links.
- L47-L48: No need to link to the EIPs here
- L49-L50: No external links.
- Throughout: Refer to EIP-3525 as "This EIP"

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Here's my first pass

File: EIPS/eip-3525.md [hunk 4]
```diff
@@ -1,54 +1,37 @@
 ---
 eip: 3525
 title: Semi-Fungible Token Standard
```
Comment (lightclient) [inline EIPS/eip-3525.md:3]: missing description field

File: EIPS/eip-3525.md [hunk 5]
```diff
@@ -1,54 +1,37 @@
...
 The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
 
-**Related Standards**
-
-- [ERC-721 Non-Fungible Token Standard](./eip-721.md)
-- [ERC-165 Standard Interface Detection](./eip-165.md)
-- [JSON Schema](https://json-schema.org/)
-- [RFC 2119 Key words for use in RFCs to Indicate Requirement Levels](https://www.ietf.org/rfc/rfc2119.txt)
-
 **Every ERC-3525 compliant contract must implement the ERC3525, ERC721 and ERC165 interfaces**
```
Comment (lightclient) [inline EIPS/eip-3525.md:35]:
```suggestion
**Every ERC-3525 compliant contract must implement the ERC-3525, ERC-721 and ERC-165 interfaces**
```

Comment (lightclient) [reply EIPS/eip-3525.md:35]: Although seems a little redundant to also declare it implements its own interface.

File: EIPS/eip-3525.md [hunk 6]
```diff
@@ -577,7 +560,7 @@ For semantic compatibility with ERC-721 as well as the flexibility of value mani
 
 ## Backwards Compatibility
 
-As mentioned in the beginning, ERC-3525 is an extension interface of ERC-721, and therefore 100% compatible with the latter.
+As mentioned in the beginning, this EIP is an compatible standard of ERC-721.
 
 ## Implementations
```
Comment (lightclient) [inline EIPS/eip-3525.md:565]: Reference Implementation

**Review by lightclient — CHANGES_REQUESTED** [review]

File: EIPS/eip-3525.md (hunk 5 above)
Comment (will-edge) [reply EIPS/eip-3525.md:35]: Not sure whether the interface name should contain '-' or not, I remember that we had removed the '-' sometime according to EIP standard.

File: EIPS/eip-3525.md [hunk 7]
```diff
@@ -577,9 +561,9 @@ For semantic compatibility with ERC-721 as well as the flexibility of value mani
 
 ## Backwards Compatibility
 
-As mentioned in the beginning, ERC-3525 is an extension interface of ERC-721, and therefore 100% compatible with the latter.
+As mentioned in the beginning, this EIP is an compatible standard of ERC-721.
 
-## Implementations
+## Reference Implementation
 
 - [ERC-3525 implementation by SOLV Protocol](https://github.com/solv-finance/erc-3525)
```
Comment (Pandapip1) [inline EIPS/eip-3525.md:568]: No external links

File: EIPS/eip-3525.md (hunk 5 above)
Comment (Pandapip1) [reply EIPS/eip-3525.md:36]: If you're referring to the EIP, you should have the `-`

File: EIPS/eip-3525.md [hunk 8]
```diff
@@ -1,54 +1,38 @@
 ---
 eip: 3525
 title: Semi-Fungible Token Standard
```
Comment (SamWilsn) [inline EIPS/eip-3525.md:3]: We try to discourage the word "standard" in titles and descriptions. Every EIP ends up being a standard, so it's a bit redundant.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Submodules should not be added to this repository.

File: EIPS/eip-3525.md [hunk 9]
```diff
@@ -478,35 +462,75 @@ This is the "ERC-3525 Metadata JSON Schema for contractURI()" referenced above.
...
-      "description": "Arbitrary properties. Values may be strings, numbers, objects or arrays."
     }
   }
 }
 ```
 
-This is the "ERC-3525 Metadata JSON Schema for slotURI(uint)" referenced above.
+This is the "EIP-3525 Metadata JSON Schema for slotURI(uint)" referenced above.
 
 ```json
```
Comment (Pandapip1) [inline EIPS/eip-3525.md:483]: Somewhere here the JSON broke. Please fix.

File: EIPS/eip-3525.md [hunk 10]
```diff
@@ -478,35 +462,75 @@ This is the "ERC-3525 Metadata JSON Schema for contractURI()" referenced above.
...
+          "order": {
+            "type": "integer",
+            "description": "Optional. According to the definition of `slot`, the best practice to generate the value of a slot is utilizing the `keccak256` algorithm to calculate the hash value of multi properties. In this scenario, the  `properties` field should contain and only contain all the properties that are related to value of `slot`, and this `order` property should be used to specify in what order these properties should be placed in the calculation process."
+          },
+	       "display_type": {
+            "type": "string",
+            "description": "Optional. Specifies in what form this property should be displayed."
+          }
+        }
+      {
```
Comment (MicahZoltu) [inline EIPS/eip-3525.md:526]:
```suggestion
      }
```

**Review by MicahZoltu — COMMENTED** [review]
Almost everything in assets/eip-3525 should be deleted and it should be paired down to one or maybe two files for the reference implementation and a single file for the test cases.  See below for more details.

----

The reference implementation should be as short/concise as possible and not include build infrastructure, package management, third party libraries, etc.  It doesn't have to be a complete example nor does it have to necessarily compile (though if it does that is a plus).  The idea is to make it so someone who wants to see an example of what an implementation of this EIP might look like they can.  It should focus solely on what is covered by *this* EIP and everything not specifically related to this EIP should be left out.

For dependencies on other EIPs, you don't have to include the code for them here (and in fact you shouldn't).  Instead, just reference with something like `IERC3525 is IERC165` but you don't need to actually define ERC165 as the reader can go look up the specification for that if they need to.

----

Everything in this repository must be CC0 licensed.

----

Test cases should be language agnostic and not come with build/test infrastructure.  A common way to present test cases is something like a JSON blob that has tuples of `function, inputs, expected_outputs, expected_exceptions, postconditions`.  It doesn't have to be a well defined or standardized format, though it is a bonus if one could theoretically drop the JSON blob into their test harness and run it.

**Comment by will-edge** [comment]
Close this PR for for easier modifications, will adopt all the required changes and suggestions in the future.

**Pull request closed without merging**